### DISCUSSION TITLE
Default install location??

### DISCUSSION THREAD

**Discussion by que762** [Devolutions/UniGetUI#2645] [Q&A] [answered]
Hey everyone, 
Everything is in the title.
Is there a parameter allowing to change the default install location??
I know you can do it using the install option but it require to do it with each package...

Thx guys!

**Comment by Lockszmith-GH** [comment, marked as answer]
I'm assuming you're asking about winget, that uses the default of the windows system.
You can probably ask for a feature request to have UniGetUI have preset of default settings - but that doesn't exist (to the extent of my knowledge, I'm just a user)

**Reply by marticliment** [reply]
The feature request is present, and will be implemented at some point. However, each software has its own install location

**Comment by cauamachado51** [comment]
https://github.com/marticliment/UniGetUI/issues/2789 issue
